### ISSUE TITLE
AdGuard still hangs Firefox

### ISSUE THREAD

**Issue by Impelon** [AdguardTeam/AdguardBrowserExtension#2251]
### AdGuard Extension version

4.1.52

### Browser version

Firefox 107.0.1

### OS version

elementary OS 6.1 (based on Ubuntu 20.04.5 LTS)

### What filters do you have enabled?

AdGuard Base filter

### Issue Details

The symptoms described in #2240 are still occurring for me:
With the AdGuard Extension enabled, Firefox's WebExtension process eats 100% of CPU and completely cripples the browser to the point it is no longer usable. Disabling the extensions immediately returns everything back to normal.

This is definitely a recent development; one month ago it used to work just fine.

### Additional Information

I'm using the flatpak installation of Firefox from Flathub:
https://flathub.org/apps/details/org.mozilla.firefox

**Comment by maximtop** [comment]
@Impelon Have you tried solutions from the issue you mentioned?

**Comment by Impelon** [comment]
@maximtop To be honest, no, because I saw it as being marked as solved and I did not see any suggestions besides disabling that specific filter, which isn't really easy to do because the extensions does not react to any commands as soon as you enable it.

On the other hand, you briefly mentioned there if anyone tried to reinstall the extension, which I have done now, and it has worked, probably because the filters have been redownloaded. 

Then, this is really an issue of communication. If one experiences this problem, one of the few ways to learn what it is about is to look through the list of *closed* issues on GitHub.
Since the Filters and the Extension seem to be versioned/distributed/updated independently, I can understand how an issue can be marked as resolved, but still occur with a version of the extension that is more recent than the point in time the filters have been fixed.
Still, since this seemed to affect quite a few people, maybe it would have been nice to provide an announcement somewhere to instruct them to update the filters, e.g. by pressing the button if possible, or by reinstalling the extension.
